tests/components/edl21/test_config_flow.py

```diff
@@ -0,0 +1,32 @@
+"""Tests for the EDL21 config flow."""
+import aiohttp
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.edl21.sensor import DOMAIN, CONF_SERIAL_PORT
+from homeassistant.components.hassio import HassioServiceInfo
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.const import CONF_NAME
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def test_show_config_form(hass: HomeAssistant) -> None:
```

Comment:
Please create at least one test, that tests the full configuration flow. Additionally, also assert/test the flow results (the actual data).

Reply:
checks added